official/resnet/resnet_run_loop.py

```diff
@@ -645,6 +683,15 @@ def define_resnet_flags(resnet_size_choices=None):
           'the expense of image resize/cropping being done as part of model '
           'inference. Note, this flag only applies to ImageNet and cannot '
           'be used for CIFAR.'))
+  flags.DEFINE_string(
+      name='worker_hosts', default=None,
```

Comment:
I guess the expectation is that the user would have started this program on all these hosts? If yes, should we clarify this?

Reply:
Yes, if they are launching the program directly they need to issue similar invocations at all workers.

Reply:
Should we clarify this in the docstring?

Reply:
Done.